R/pkg/R/functions.R

```diff
@@ -653,6 +653,25 @@ setMethod("hash",
             column(jc)
           })
 
+#' data_type
+#'
+#' Returns the data type of a given column.
+#'
+#' @param x Column to get the data type.
+#'
+#' @rdname data_type
+#' @name data_type
+#' @family misc functions
```

Comment:
can you follow the pattern, like for "hash" above? we don't have individual rdname etc for each function